### PR TITLE
fix typo for GetHostRootGid

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

File: namespaces/exec.go [hunk 1]
```diff
@@ -255,7 +255,7 @@ func AddUidGidMappings(sys *syscall.SysProcAttr, container *libcontainer.Config)
 //
 // console: the /dev/console to setup inside the container
 // init: the program executed inside the namespaces
-// root: the path to the container json file and information
+// dataPath: the path saving container's date
```
Comment (mrunalp) [inline namespaces/exec.go]: It isn't for date. It is for state.

Comment (mrunalp) [inline namespaces/exec.go]: I think it is a typo you meant data :)

**Comment by mrunalp** [comment]
Also, maybe squash the commits.

**Comment by hqhq** [comment]
Updated.

File: namespaces/exec.go [hunk 2]
```diff
@@ -255,7 +255,7 @@ func AddUidGidMappings(sys *syscall.SysProcAttr, container *libcontainer.Config)
 //
 // console: the /dev/console to setup inside the container
 // init: the program executed inside the namespaces
-// root: the path to the container json file and information
+// dataPath: the path saving container's states
```
Comment (mrunalp) [inline namespaces/exec.go]: Could you change it to something like
dataPath: the path to the directory under which the container's state file is stored

File: namespaces/exec.go [hunk 3]
```diff
@@ -294,11 +294,9 @@ func DefaultCreateCommand(container *libcontainer.Config, console, dataPath, ini
 // DefaultSetupCommand will return an exec.Cmd that joins the init process to set it up.
 //
 // console: the /dev/console to setup inside the container
+// dataPath: the path saving container's states
```
Comment (mrunalp) [inline namespaces/exec.go]: Same here

Comment (hqhq) [inline namespaces/exec.go]: Updated.

**Comment by mrunalp** [comment]
@hqhq Thanks! LGTM

**Comment by rjnagal** [comment]
LGTM.

Thanks for the cleanup.

**Pull request merged**